src/main/java/ua/tc/marketplace/util/mapper/AttributeMapper.java

```diff
@@ -0,0 +1,15 @@
+package ua.tc.marketplace.util.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import ua.tc.marketplace.config.MapperConfig;
+import ua.tc.marketplace.model.dto.ad.AdAttributeDto;
+import ua.tc.marketplace.model.entity.AdAttribute;
+
+/** Mapper interface for converting AdAttribute entities to AdAttributeDto. */
+@Mapper(config = MapperConfig.class)
+public interface AttributeMapper {
+
+  @Mapping(target = "name", source = "attribute.name")
+  AdAttributeDto adAttributeDto(AdAttribute adAttribute);
```

Comment:
there is no action seen in such a name of the method. I would name it shortly toDto. But i guess we should discuss and vote for the most appropriate variant